### DISCUSSION TITLE
第5节课作业(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#313] [作业]
请在下方提交作业链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/7bb96bd7-c4e6-45b5-955e-de1515c6e49b

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/homework05.md

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework4_LMDeploy.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/homework_lmdeploy/blob/main/homework5.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135624316

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135599186

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course5_hw.md

**Comment by Alias-z** [comment]
388+https://github.com/Alias-z/InternLM-Camp-2024/blob/main/Assignment%204%20LLMDeploy/README.md

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson5/assign.md

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/CMKXdI5oOopi1xx6cdjcOrKhnHK?from=from_copylink

**Comment by 00z768** [comment]
1095 + https://app.yinxiang.com/Home.action?referralCode=ebcc-organic-yxbj&login=true#n=8b6628d4-7e22-42ce-b21a-22111c48cfae&s=s21&ses=4&sh=2&sds=5&

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/5homework.md

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/HWDay05.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Five-lmdeploy/Task.md

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9392601
